### PR TITLE
Bug Fixes and Updated Color Scheme

### PR DESCRIPTION
**Pull request by 07jasjeet**
Expanded material3 and fixed some bugs.

### Expansion:
1) Added mini brainzPlayer's colors to ListenBrainzTheme and mentioned at appropriate places.
2) Fixed status bar foreground [MOBILE-69]
3) Updated system navigation background and foreground.
4) Replaced onAttachFragment (deprecated function) with documentation recommend function.

### Bug 1: 
When we open the app (More visible in dark mode), the back layer content takes time to load which causes that area to be white for some time (no matter how long we keep the splash screen).
[White Background Flicker.webm](https://user-images.githubusercontent.com/98077881/205621979-f7c0ce18-210b-461c-b4c7-70e8657d595a.webm)

**After fix**

[After Flicker Fix.webm](https://user-images.githubusercontent.com/98077881/205624042-4882f97d-0516-4b70-bd96-1b316fe9ffb0.webm)

### Bug 2:
Back layer content was not taking full size of screen causing the brainzPlayer's PagerView to extend to the top.

**After Fix**

### CODE REVIEW

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerActivity.kt [hunk 1]
```diff
@@ -73,7 +73,7 @@ class BrainzPlayerActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            ListenBrainzTheme {
+            ListenBrainzTheme(window = window) {
```
Comment (akshaaatt) [inline app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerActivity.kt:76]: Why has this been added?

File: app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt [hunk 2]
```diff
@@ -27,7 +30,7 @@ import org.listenbrainz.android.presentation.theme.ListenBrainzTheme
 
 @AndroidEntryPoint
 class DashboardActivity : ComponentActivity() {
-
+    @RequiresApi(Build.VERSION_CODES.TIRAMISU)
```
Comment (akshaaatt) [inline app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt:33]: Do we want to do this?

**Review by akshaaatt — APPROVED** [review]
Awesome work! Looks good to me so far 💯

File: app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt (hunk 2 above)
Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt:33]: "android.Manifest.permission.READ_MEDIA_AUDIO" permission in line 45 shows lint warning if we remove this.
Although this being here doesn't interrupt with my android 10 device so I think it should be fine

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerActivity.kt (hunk 1 above)
Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerActivity.kt:76]: Current I couldn't find a Providable Composition Local to access the current window that the composable is being used in.
This parameter is being used to set the status bars and system navigations theming of the given window.

File: app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt (hunk 2 above)
Comment (akshaaatt) [reply app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt:33]: Thanks for the clarification.

**Pull request merged**